src/js/utils/obj.js

```diff
@@ -30,3 +31,55 @@ export function reduce(object, fn, initial = 0) {
     (accum, key) => fn(accum, object[key], key),
     initial);
 }
+
+/**
+ * Object.assign-style object shallow merge/extend.
+ *
+ * @param  {Object} target
+ * @param  {Object} ...sources
+ * @return {Object}
+ */
+export function assign(target, ...sources) {
```

Comment:
However, there's one nice usecase of not forcing target to be an empty object:
```js
Object.assign(el.style, {
  'color': 'blue',
  'background': 'red'
});
```
Or similar. But yeah, *most* times, it probably does want to just be an empty object.